TACC/Core-Portal
# task/FP-1037 - Management commands for forcing system creation

## Overview: ##

Two management commands:

For forcing system creation:
`python manage.py force_system_creation --username username`

(this does not set onboarding states, it only activates the abaco actor for given systems, bypassing the system integrity check.)

For rewriting existing systems to cloud.corral:
`python manage.py cloud_corral --system corral.home.username`

## Related Jira tickets: ##

* [FP-1037](https://jira.tacc.utexas.edu/browse/FP-1037)

## Summary of Changes: ##

## Testing Steps: ##
1. Confirm your home systems already work
2. Run `./manage.py force_system_creation --username $USERNAME` for your username
3. If you have frontera home as a system, you should see that you need to push keys again, and should see that the system's `lastModified` param has updated.
